core/domain/suggestion_services.py

```diff
@@ -236,7 +279,11 @@ def accept_suggestion(suggestion, reviewer_id, commit_message, review_message):
         Exception: The commit message is empty.
     """
     if suggestion.is_handled:
-        raise Exception('The suggestion has already been accepted/rejected.')
+        raise Exception(
+            'The suggestion with id %s has already been accepted/rejected.' % (
+                suggestion.suggestion_id
```

Comment:
suggestion.suggestion_id**))** (to reduce extra lines and lots of line with parenthesis, maybe? elsewhere as well)

Reply:
Just to confirm -- either convention is fine. The rule that @kaylahardie is following is to have the closing paren on the same indentation level as the open paren, and that's OK too.